src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -100,7 +100,8 @@ Value mapToLowerScalarOp<ONNXSinhOp>(Operation *op, ArrayRef<Type> result_types,
   // ONNXSinhOp(%X) = DivFOp(SubFOp(ExpOp(%X), ExpOp(NegFOp(%X))),
   //                         ConstantOp 2)
   auto loc = op->getLoc();
-  Value operand = operands[0];
+  ONNXAsinhOpOperandAdaptor operandAdaptor(operands);
+  Value operand = operandAdaptor.input();
```

Comment:
Sorry for the confusion in using the same name. I just want to say that operands[0] here is NOT the first operand of the ONNX Op, but the operand of the function mapToLowerScalarOp. In particular, operands[0] here is expected to be a scalar value while `operandAdaptorl.input()` is a tensor value. Hence, we cannot replace it with the ONNX Op's operand. I wonder why our tests did not detect this error.

Reply:
@tungld thanks for spotting this mistake! Though I'm not surprised it made past the end-to-end tests, given that the adaptor will return the first element of a list of operands, which corresponds to the value `input` getter is supposed to return, given the correct list of operand inputs.

Reply:
You are right, the code pattern is so alike, it really required to pay attention to how the functions were called. Thanks for spotting this. Did run all the test including e2e, so maybe the input() or data() were by chance [0].